docs/README.md

```diff
@@ -1150,6 +1154,155 @@ In this setup there is an external control plane cluster (`cluster1`) and a remo
+IPv4 and IPv6 traffic. With many cloud providers also beginning to offer dual-stack Kubernetes clusters, it's easier
+than ever to run services that function across both address types. Istio introduced dual-stack as an experimental
+feature in version 1.17, and it's expected to be promoted to [Alpha](https://github.com/istio/istio/issues/47998) in
+version 1.24. With Istio in dual-stack mode, services can communicate over both IPv4 and IPv6 endpoints, which helps
+organizations transition to IPv6 while still maintaining compatibility with their existing IPv4 infrastructure.
+
+When Kubernetes is configured for dual-stack, it automatically assigns an IPv4 and an IPv6 address to each pod,
+enabling them to communicate over both IP families. For services, however, you can control how they behave using
+the `ipFamilyPolicy` setting.
+
+Service.Spec.ipFamilyPolicy can take the following values
+- SingleStack: Only one IP family is configured for the service, which can be either IPv4 or IPv6.
+- PreferDualStack: Both IPv4 and IPv6 cluster IPs are assigned to the Service when dual-stack is enabled.
+                   However, if dual-stack is not enabled or supported, it falls back to singleStack behavior.
+- RequireDualStack: The service will be created only if both IPv4 and IPv6 addresses can be assigned.
+
+This allows you to specify the type of service, providing flexibility in managing your network configuration.
+For more details, you can refer to the Kubernetes [documentation](https://kubernetes.io/docs/concepts/services-networking/dual-stack/#services).
+
+### Prerequisites
+
+- Kubernetes 1.23 or later configured with dual-stack support.
+- Sail Operator is installed.
+- Kind cluster with dual-stack networking.
+
+### Installation Steps
+
+You can use any existing Kind cluster that supports dual-stack networking or, alternatively, install one using the following command.
+
+```sh
+kind create cluster --name istio-ds --config - <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+ ipFamily: dual
+EOF
+```
+
+Note: If you installed the KinD cluster using the command above, install the [Sail Operator](#getting-started) before proceeding with the next steps.
+
+1. Create the `Istio` resource with dual-stack configuration.
+
+   ```sh
+   kubectl get ns istio-system || kubectl create namespace istio-system
+   kubectl apply -f - <<EOF
+   apiVersion: sailoperator.io/v1alpha1
+   kind: Istio
+   metadata:
+     name: default
+   spec:
+     values:
+       meshConfig:
+         defaultConfig:
+           proxyMetadata:
+             ISTIO_DUAL_STACK: "true"
+       pilot:
+         ipFamilyPolicy: RequireDualStack
+         env:
+           ISTIO_DUAL_STACK: "true"
+     version: v1.23.2
+     namespace: istio-system
+   EOF
+   kubectl wait --for=jsonpath='{.status.revisions.ready}'=1 istios/default --timeout=3m
+   ```
+
+2. If running on OpenShift platform, create the IstioCNI resource as well.
```

Comment:
I guess technically we also require setting the `openshift` profile but we're working on auto-detection so it's okay if we leave that out